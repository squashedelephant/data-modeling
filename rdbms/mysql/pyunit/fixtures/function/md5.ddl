CREATE TABLE data(id INT NOT NULL AUTO_INCREMENT, word CHAR(20), hash CHAR(32), PRIMARY KEY (id));
