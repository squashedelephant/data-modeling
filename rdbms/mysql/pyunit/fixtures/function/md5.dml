SET @word = 'special';
INSERT INTO data(word, hash) VALUES(@word, MD5(@word));
