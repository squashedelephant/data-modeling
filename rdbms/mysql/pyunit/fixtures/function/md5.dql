SELECT hash FROM data WHERE word = 'special';
