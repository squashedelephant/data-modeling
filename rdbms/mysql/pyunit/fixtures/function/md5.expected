0bd6506986ec42e732ffb866d33bb14e
